functional/MockitoTests/playlist.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<!--
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+-->
+
+<playlist xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../../../TKG/resources/playlist.xsd">
+	<test>
+		<testCaseName>MockitoMockTest</testCaseName>
+		<disables>
+			<disable>
+				<comment>Temporarily disabled on z/OS for openj9/issues/19331</comment>
```

Comment:
Please use the direct link
`<comment>https://github.com/eclipse-openj9/openj9/issues/19331</comment>`